library/src/main/res/values/attrs.xml

```diff
@@ -7,6 +7,7 @@
     <attr name="tsquare_dayTextColor" format="color"/>
     <attr name="tsquare_titleTextColor" format="color"/>
     <attr name="tsquare_displayHeader" format="boolean"/>
+    <attr name="tsquare_displayWeekView" format="boolean"/>
```

Comment:
let's go ahead and rename all references to "weekView" too, in the attrs and the code

Reply:
sounds good! on it!